check-certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+AGENT_CONFIG="/mnt/data/etc/wb-cloud-agent.conf"
+
+print_bundle_part() {
+    awk -v "req_part=$1" 'BEGIN { c = 0; } /BEGIN CERT/{c++} c == req_part { print }'
```

Comment:
В awk переменная по дефолту и так будет 0:
```suggestion
    awk -v "req_part=$1" '/BEGIN CERT/{c++} c == req_part { print }'
```
См. https://www.gnu.org/software/gawk/manual/html_node/Using-Variables.html#Using-Variables-in-a-Program

Reply:
это копипаста, если что

Reply:
готово